src/distortion.js

```diff
@@ -76,9 +139,10 @@ class Distortion extends Effect {
    *
    * @method process
    * @for p5.Distortion
-   * @param {Number} [amount=0.25] Unbounded distortion amount.
+   * @param {Object} Signal  An object that outputs audio
```

Comment:
Do you think we can match the name here to the variable name on line 147? Either by calling this `src`, or by renaming the parameter to `signal` (lowercase, to follow variable naming conventions)

Reply:
yes, I think `src` is better, I copied it from the [filter](https://p5js.org/reference/#/p5.Filter/process) and [delay](https://p5js.org/reference/#/p5.Delay/process) documentation. I think those 2 documentations also need that change